### PR TITLE
feature(unlock-protocol-com): adding base blog post

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding the base blog post.

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/blog/base.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Base
```
Comment (ccarfi) [inline unlock-protocol-com/blog/base.md:2]:
```suggestion
title: Unlock Protocol has been deployed to the Base L2 🔵
```

File: unlock-protocol-com/blog/base.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+[Coinbase](https://www.coinbase.com/home) (one of our investors) has undeniably contributed massively to the crypto ecosystem, and we believe Base represents a step in democratizing web3 and the onchain web.
+
+As an L2, Base benefits from the “security” budget of Ethereum, while also lowering transaction costs.
+
+## How does it work?
+
+Deploying a membership contract to Base follows the same process as deploying to any other of networks supported by Unlock Protocol. You can deploy your lock(s) using the Unlock Protocol Dashboard (or do it through code) and then use any of the tools available in the Unlock Protocol ecosystem, including using our checkout to sell your NFT memberships, airdrop them, or tap into any of the other capabilities available to the community.
+
+(links to other resources TBD)
```
Comment (ccarfi) [inline unlock-protocol-com/blog/base.md:19]: @julien51 what's the ETA for the docs to be live so they can be linked to here?

Comment (julien51) [reply unlock-protocol-com/blog/base.md:19]: It's live in the docs. https://docs.unlock-protocol.com/core-protocol/unlock/networks 
Do we want to add instructions on how to "add" Base to your wallet + bridge tokens? I can do that.

Comment (ccarfi) [reply unlock-protocol-com/blog/base.md:19]: per today's standup, docs are live (networks page on docs)

File: unlock-protocol-com/blog/base.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Unlock Protocol has been deployed to the Base L2 🔵
+authorName: Julien Genestoux
+publishDate: August 2, 2023
```
Comment (ccarfi) [inline unlock-protocol-com/blog/base.md:4]:
```suggestion
publishDate: August 9, 2023
```

File: unlock-protocol-com/blog/base.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Unlock Protocol has been deployed to the Base L2 🔵
```
Comment (ccarfi) [inline unlock-protocol-com/blog/base.md:2]:
```suggestion
title: Unlock Protocol Has Been Deployed to the Base L2 🔵
```

File: unlock-protocol-com/blog/base.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+title: Unlock Protocol Has Been Deployed to the Base L2 🔵
+authorName: Julien Genestoux
+publishDate: August 9, 2023
+description: You can use any of the tools available in the Unlock Protocol ecosystem to create membership smart contracts or ticket events on Base.
+image: https://docs.base.org/img/logo.svg
```
Comment (ccarfi) [inline unlock-protocol-com/blog/base.md:6]: @julien51 hotlinking to external image risks breaking if/when they change that image (probably not the case here, but have also seen folks block external references to images). safer to refer to an image on our domain.

Comment (julien51) [reply unlock-protocol-com/blog/base.md:6]: Good point1 Let me update that!

**Pull request merged**